### DISCUSSION TITLE
Feature for Custom Shortcuts

### DISCUSSION THREAD

**Discussion by raghuvanshaj** [dailydotdev/daily#901] [Feature Requests]
Hey , I am recently using the daily.dev chrome extension and I am very happy with it . The only thing I find missing is to improve the shortcut UI . It would be very helpful if you can improve UI for Custom Shortcut as sometimes it is a bit difficult to recognise the one with small logos ....

**Comment by idoshamun** [comment]
Hi @agroot2002,

Thanks for the suggestion, we'll consider it :)
